docs/content/posts/2025-01-18-release-cdc-3.3.0.md

```diff
@@ -0,0 +1,79 @@
+---
+title:  "Apache Flink CDC 3.3.0 Release Announcement"
+date: "2025-01-18T08:00:00.000Z"
+authors:
+- ruanhang:
+  name: "Hang Ruan"
+aliases:
+- /news/2025/01/18/release-cdc-3.3.0.html
+---
+
+The Apache Flink Community is excited to announce the release of Flink CDC 3.3.0!
+This release introduces more features in transform and connectors and improve 
+usability and stability of existing features.
+
+Flink CDC release packages are available at [Releases Page](https://flink.apache.org/downloads.html#flink-cdc),
+and documentations are available at [Flink CDC documentation](https://nightlies.apache.org/flink/flink-cdc-docs-release-3.3) page.
+Looking forward to any feedback from the community through the Flink [mailing lists](https://flink.apache.org/community.html#mailing-lists) or [JIRA](https://issues.apache.org/jira/browse/flink)!
+
+# Highlights
+
+## Connectors
+
+### New Pipeline Connectors
+
+Flink CDC 3.3.0 introduces 2 new pipeline connector:
+
+* OceanBase sink
+* MaxCompute sink 
+
+### MySQL
+
+* Support parsing gh-ost generated schema changes.
+* Parse array-typed key index binlog created between 8.0.17 and 8.0.18 rightly.
+* Support passing op_ts to meta field in Event.
+* Fixed deadlock after adding new tables.
+* Fixed a bug that works with FLOAT type with precision
+
+### Apache Paimon
+
+* Apply default value options when apply add column change.
+* Reuse sequence number when schema evolution happened.
+* Fixed commit conflict problem in PaimonSink.
+* Remove Catalog.ColumnAlreadyExistException when apply applyAddColumnEventWithPosition in paimon.
```

Comment:
Bump Paimon version to 0.9.0.

Added in:
https://issues.apache.org/jira/browse/FLINK-36524